### PR TITLE
I18n branches

### PR DESCRIPTION
**Pull request by daschuer**
This uses the newly created https://www.transifex.com/mixxx-dj-software/mixxx-dj-manual-23 Project on Transiflex to avid conflicts with the the main branch. This is on top of https://github.com/mixxxdj/manual/pull/503 which should be merged first.

### CODE REVIEW

**Comment by Swiftb0y** [comment]
please rebase

File: .tx/config [hunk 1]
```diff
@@ -1,139 +1,137 @@
...
-type = PO
```
[the remark this answers is not in the document]
Comment (daschuer) [reply .tx/config:4]: I have recreated the file using the tx tools and it was not added. 
Do you know the purpose of it? Is it still necessary? 
I would prefer not to maintain a list of languages.

**Comment by daschuer** [comment]
> please rebase

Done

File: .tx/config (hunk 1 above)
[the remark this answers is not in the document]
Comment (Swiftb0y) [reply .tx/config:4]: >  Do you know the purpose of it? Is it still necessary?

Unfortunately not. Does anyone?

[the remark this answers is not in the document]
Comment (ronso0) [reply .tx/config:4]: @sblaisot maybe?

[the remark this answers is not in the document]
Comment (daschuer) [reply .tx/config:4]: Here is the description: 
https://docs.transifex.com/client/client-configuration#language-mappings

It looks like it was done to to map the _ from transiflex to to - with Sphinx.  

We have on the other hand a warning that Sphinx is complaining about exactly this dash: 
```
WARNING: reading error: /home/runner/work/manual/manual/source/locale/zh-TW/LC_MESSAGES/sphinx.po, expected only letters, got 'zh-tw'
```
strange

[the remark this answers is not in the document]
Comment (daschuer) [reply .tx/config:4]: Sphinx also uses the _ : 
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language

And I can confirm that de_DE works. 

I guess there is a relates issue and this PR will cause may regression. But we need to find out. 
At least we are on the correct rout to fix this issue.

**Comment by daschuer** [comment]
Now it should work. 
Next we should get rid of "supported_languages" in [../conf.py](https://github.com/mixxxdj/manual/pull/504/commits/a800ec03871a72cdd6fbb8e07ce22b6726119a48)

**Comment by daschuer** [comment]
> Next we should get rid of "supported_languages" in [../conf.py](https://github.com/mixxxdj/manual/pull/504/commits/a800ec03871a72cdd6fbb8e07ce22b6726119a48)

I have looked into it, probably not worth the hassle. 
For now it is ready to merge. 
@Swiftb0y do you agree?

**Comment by daschuer** [comment]
@Holzhaus This branch moves from de-de to de_DE. My local test where successful. The only question is whether the Netlify redirect rules are independent against the _ vs - language format. The Do

I have no Idea how to test it, so I think we can just merge it and fix the redirect rule immediately if it tuns out to be broken. 
The file of question is this https://github.com/mixxxdj/manual/blob/2.3/build_html.sh

**Comment by daschuer** [comment]
Juchhu! This PR also fixes the issue that the https://downloads.mixxx.org/manual/2.3/mixxx-manual-2.3-de_DE.pdf manual is translated and not untranslated like https://downloads.mixxx.org/manual/2.3/mixxx-manual-2.3-de-DE.pdf

**Pull request merged**